nym-vpn-lib/src/credentials.rs

```diff
@@ -39,3 +44,128 @@ pub async fn import_credential_file(credential_file: PathBuf, data_path: PathBuf
     let raw_credential = fs::read(credential_file)?;
     import_credential(raw_credential, data_path).await
 }
+
+pub async fn check_imported_credential(data_path: PathBuf, gateway_id: &str) -> anyhow::Result<()> {
```

Comment:
but this won't necessarily check the latest imported credential. because `fetch_valid_credential` will call `get_next_unspent_credential` which, from what i recall, will get a credential with lowest id